api_intro.py

```diff
@@ -1,18 +1,25 @@
 from datetime import date
+
 import requests
 
 import config
 
 # Client IDs and Secrets from config.py
-ob_client_id = config.OB_CLIENT_ID
-ob_client_Secret = config.OB_CLIENT_SECRET
-sn_client_id = config.SN_CLIENT_ID
-sn_client_Secret = config.SN_CLIENT_SECRET
+try:
+    OB_CLIENT_ID = config.OB_CLIENT_ID
+    OB_CLIENT_SECRET = config.OB_CLIENT_SECRET
+    SN_CLIENT_ID = config.SN_CLIENT_ID
+    SN_CLIENT_SECRET = config.SN_CLIENT_SECRET
+    BASE_URL = config.BASE_URL
+except AttributeError:
+    raise Exception('Config file is missing configuration.')
```

Comment:
That works. However, the config file is usually a json or a yaml file. Based on what you programed right now, if a user didn't provide a `config.py` file, it won't ever run into this try-catch block since it will be exited at line 5 since config module is not defined. For better practice, I'd recommend doing the following:

1. You can still have a file called config.py, and it will take care of opening and reading attributes from a config file (either in json or yaml), as well as error handling e.g. config file not valid or required attributes not defined, etc.
2. Import the config.py to api_intro.py and read the loaded config data.

Reply:
I changed the script to use a json file instead. Let me know if I did this correctly.